docs/v3.x/getting-started/svelte.md

```diff
@@ -0,0 +1,533 @@
+		restaurants = res.data
+	} catch (e) {
+		error = e
+	}
+});
+
+</script>
+
+{#if error !== null}
+  {error}
+{:else}
+  <ul>
+  {#each restaurants as restaurant}
+    <li>
+  		{restaurant.name}
+    </li>
+  {/each}
+  </ul>
+{/if}
+```
+
+:::
+
+::: tab fetch
+
+`./src/App.svelte`
+
+```js
+<script>
+import { onMount } from 'svelte';
+
+let restaurants = [];
+let error = null
+
+onMount(async () => {
+	const parseJSON = (resp) => (resp.json ? resp.json() : resp);
+	const checkStatus = (resp) => {
+    if (resp.status >= 200 && resp.status < 300) {
+      return resp;
+    }
+    return parseJSON(resp).then((resp) => {
+      throw resp;
+    });
+  };
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+
+	try {
+		const res = await fetch("http://localhost:1337/restaurants", {
+		  method: "GET",
+		  headers: {
+		     'Content-Type': 'application/json'
+		  },
+		}).then(checkStatus)
+      .then(parseJSON);
+		restaurants = res
+	} catch (e) {
+		error = e
+	}
+});
+</script>
+
+{#if error !== null}
+  {error}
+{:else}
+  <ul>
+  {#each restaurants as restaurant}
+    <li>
+  		{restaurant.name}
+    </li>
+  {/each}
+  </ul>
+{/if}
+```
+
+:::
+
+::::
+
+### POST Request your collection type
+
+Execute a POST request on the `restaurant` Collection Type in order to create a restaurant.
+
+Be sure that you activated the `create` permission for the `restaurant` Collection Type and the `find` permission fot the `category` Collection type.
+
+In this example a `japanese` category has been created which has the id: 3
+
+:::: tabs
+
+::: tab axios
+
+_Request_
+
+```js
+import axios from 'axios';
+
+axios
+  .post('http://localhost:1337/restaurants', {
+    name: 'Dolemon Sushi',
+    description: 'Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious',
+    categories: [3],
+  })
+  .then(response => {
+    console.log(response);
+  });
+```
+
+:::
+
+::: tab fetch
+
+_Request_
+
+```js
+fetch('http://localhost:1337/restaurants', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+  },
+  body: JSON.stringify({
+    name: 'Dolemon Sushi',
+    description: 'Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious',
+    categories: [3],
+  }),
+})
+  .then(response => response.json())
+  .then(data => console.log(data));
+```
+
+:::
+
+_Response_
+
+```json
+{
+  "id": 2,
+  "name": "Dolemon Sushi",
+  "description": "Unmissable Japanese Sushi restaurant. The cheese and salmon makis are delicious",
+  "created_by": null,
+  "updated_by": null,
+  "created_at": "2020-08-04T09:57:11.669Z",
+  "updated_at": "2020-08-04T09:57:11.669Z",
+  "categories": [
+    {
+      "id": 3,
+      "name": "Japanese",
+      "created_by": 1,
+      "updated_by": 1,
+      "created_at": "2020-07-31T11:36:23.164Z",
+      "updated_at": "2020-07-31T11:36:23.172Z"
+    }
+  ]
+}
+```
+
+::::
+
+### Example
+
+:::: tabs
+
+::: tab axios
+
+`./src/App.svelte`
+
+```js
+<script>
```

Comment:
Same here about the example. Why don't we have it for the other apps?

Reply:
Difficult to reproduce on specific tech I'm not comfortable with

Reply:
Would love to help here also but I'm not familiar with svelte either